ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -10,6 +10,37 @@ public protocol BottomCommandingControllerDelegate: AnyObject {
 
     /// Called when `collapsedHeightInSafeArea` changes.
     @objc optional func bottomCommandingControllerCollapsedHeightInSafeAreaDidChange(_ bottomCommandingController: BottomCommandingController)
+
+    /// Called after the bottom sheet expansion state changes.
+    ///
+    /// External changes to `isHidden` will not trigger this callback.
+    /// - Parameters:
+    ///   - bottomCommandingController: The caller object.
+    ///   - expansionState: The expansion state the sheet moved to.
+    ///   - commandingInteraction: If the state change was caused by user interaction, it will be indicated using this enum.
+    ///   - sheetInteraction: If `commandingInteraction` is `.sheetInteraction`, this enum will contain more information about what triggered the state change.
+    @objc optional func bottomCommandingController(_ bottomCommandingController: BottomCommandingController, sheetDidMoveTo expansionState: BottomSheetExpansionState, commandingInteraction: BottomCommandingInteraction, sheetInteraction: BottomSheetInteraction)
```

Comment:
minor: Even if swift-lint doesn't force us to, I like the cleanliness of putting one param per line when you have this many params. Separately, we should make swift-lint force us to do so and/or just switch to swift-format